packages/react-components/src/organisms/ResearchOutputFormSharingCard.tsx

```diff
@@ -203,14 +203,13 @@ const ResearchOutputFormSharingCard: React.FC<
           placeholder="Select subtype"
         />
       )}
-      <LabeledTextArea
+      <LabeledTextEditor
         title="Description"
         subtitle="(required)"
         tip="Add an abstract or a summary that describes this work."
         onChange={onChangeDescription}
         getValidationMessage={() => 'Please enter a description'}
         required
-        enabled={!isSaving}
         value={descriptionMD}
         info={
```

Comment:
I don't think we need this info anymore 😄 
<img width="938" alt="Screenshot 2024-10-14 at 06 36 16" src="https://github.com/user-attachments/assets/707078b4-cbf2-45de-899f-f9b3f93e12d4">